packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -50,184 +64,271 @@ describe("toMapTree", () => {
 	});
 
 	it("handle", () => {
-		const schemaBuilder = new SchemaBuilder({ scope: "test" });
-		const schema = schemaBuilder.intoSchema(schemaBuilder.handle);
+		const schemaFactory = new SchemaFactory("test");
+		const schema = schemaFactory.handle;
 
 		const tree = new MockHandle<string>("mock-fluid-handle");
 
-		const actual = nodeDataToMapTree(tree, schema, schema.rootFieldSchema.allowedTypeSet);
+		const actual = nodeDataToMapTree(tree, [schema]);
 
 		const expected: MapTree = {
-			type: leaf.handle.name,
+			type: brand(schemaFactory.handle.identifier),
 			value: tree,
 			fields: new Map(),
 		};
 
 		assert.deepEqual(actual, expected);
 	});
 
-	it("list (non-empty)", () => {
-		const schemaBuilder = new SchemaBuilder({ scope: "test" });
-		const rootSchema = schemaBuilder.list("list", [schemaBuilder.number, schemaBuilder.handle]);
-		const schema = schemaBuilder.intoSchema(rootSchema);
+	describe("list", () => {
+		it("Non-empty list", () => {
```

Comment:
this test seems like a little more than a non-empty list. (also name is stale: should be azrrayNode not list).

If you need to test nesting or polymorphisc lists, that seems like it should be its own unit test

Reply:
Fair. Since I didn't update the tests' structures here (beyond some basic refactoring), I think I'll save that for a subsequent PR. But I'll file a task for that,

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/7597